### PR TITLE
Structure_oM: add contour and geometric line loads

### LINKED ISSUE #534: Structure_oM: Add Contour and Geometric Line loads
#### Description:

Both load types have been added as prototypes in Robot_Toolkit and have been proven useful.

Both should be migrated to the Structure_oM to allow them to be used in the wider framework.

Links to oM files:

https://github.com/BHoM/Robot_Toolkit/blob/master/Robot_oM/Structural/Loads/ContourLoad.cs

https://github.com/BHoM/Robot_Toolkit/blob/master/Robot_oM/Structural/Loads/GeometricalLineLoad.cs

### PR DESCRIPTION
**Pull request by JosefTaylor**
### Issues addressed by this PR

Closes #534

### CODE REVIEW

File: Structure_oM/Loads/ContourLoad.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2018, the respective contributors. All rights reserved.
```
Comment (FraserGreenroyd) [inline Structure_oM/Loads/ContourLoad.cs]: Copyright should be 2019

File: Structure_oM/Loads/GeometricalLineLoad.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2018, the respective contributors. All rights reserved.
```
Comment (FraserGreenroyd) [inline Structure_oM/Loads/GeometricalLineLoad.cs]: Copyright should be 2019

File: Structure_oM/Loads/ContourLoad.cs (hunk 1 above)
Comment (JosefTaylor) [reply Structure_oM/Loads/ContourLoad.cs]: See commit- copyright 2019 (or is it 2015-2019?)

**Comment by FraserGreenroyd** [comment]
Thanks @JosefTaylor I've dismissed my review so that this can be approved as appropriate by @IsakNaslundBh or @rwemay :smile:

**Comment by JosefTaylor** [comment]
Ready for @IsakNaslundBh and @rwemay to review!

**Pull request merged**